fixture/package.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cargo build --release --target wasm32-unknown-unknown -p integrity
+cargo build --release --target wasm32-unknown-unknown -p coordinator1
+cargo build --release --target wasm32-unknown-unknown -p coordinator2
+
+pushd package/dna1 || exit 1
+hc dna pack .
+popd || exit 1
+
+pushd package/dna2 || exit 1
+hc dna pack .
+popd || exit 1
+
+pushd package/happ1 || exit 1
+hc app pack .
+popd || exit 1
+
+pushd package/happ2 || exit 1
+hc app pack .
+popd || exit 1
```

Comment:
Nitpicks:
1. The cargo build lines could be combined by adding multiple `-p` flags.
2. This script, as is, needs to be run from inside the fixtures directory, but you can use `dirname $0` to make it runnable anywhere.

```suggestion
SCRIPT_DIR=$(dirname "$0")

cargo build --manifest-path "$SCRIPT_DIR/Cargo.toml" --release --target wasm32-unknown-unknown -p integrity -p coordinator1 -p coordinator2

pushd "$SCRIPT_DIR/package/dna1" || exit 1
hc dna pack .
popd || exit 1

pushd "$SCRIPT_DIR/package/dna2" || exit 1
hc dna pack .
popd || exit 1

pushd "$SCRIPT_DIR/package/happ1" || exit 1
hc app pack .
popd || exit 1

pushd "$SCRIPT_DIR/package/happ2" || exit 1
hc app pack .
popd || exit 1

```

Reply:
Ah, well done, GitHub. Great diff :roll_eyes:

Reply:
Nice, yes that works well thank you :) Also tweaked CI to call the script from the root of the repository